pkg/driver/controller.go

```diff
@@ -437,9 +437,22 @@ func (d *controllerService) CreateSnapshot(ctx context.Context, req *csi.CreateS
 		klog.V(4).Infof("Snapshot %s of volume %s already exists; nothing to do", snapshotName, volumeID)
 		return newCreateSnapshotResponse(snapshot)
 	}
+
+	snapshotTags := map[string]string{
+		cloud.SnapshotNameTagKey: snapshotName,
+	}
+	if d.driverOptions.kubernetesClusterID != "" {
```

Comment:
Can you also update the usage string of `k8s-tag-cluster-id` in `controller_options.go`?

Reply:
I removed the sentence about feature-parity with the in-tree driver since the in-tree driver does not tag snapshots. Let me know if the new sentence is sufficient.